databases/week2/school_database.sql

```diff
@@ -0,0 +1,29 @@
+CREATE DATABASE school;
+SET NAMES utf8mb4;
+
+USE school;
+CREATE TABLE `Class` (
+    `ClassId` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
It coud be just 'id' here since it's an id of an entry and a primary key in this table. Naming it ClassId would sometimes get others confused 🤓